Week1/homework/src/results.js

```diff
@@ -0,0 +1,181 @@
+'use-strict';
+
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'world',
+});
+
+function one() {
+  connection.connect();
+  connection.query(
+    'SELECT name, population FROM countries WHERE population > 8000000 LIMIT 20',
+    function(error, results) {
+      if (error) throw error.message;
+      console.log(`
+        1- Countries with population greater than 8 million:
+        `);
+      results.forEach(country => console.log(JSON.parse(JSON.stringify(country))));
+      console.log(`
+        `);
+    }
+  );
+  connection.end();
+}
+
+function two() {
+  connection.connect();
+  connection.query('SELECT name FROM countries WHERE name LIKE "%land%" LIMIT 20', function(
+    error,
+    results
+  ) {
+    if (error) throw error.message;
+    console.log(`
+        2- Countries that have “land” in their names:
+        `);
+    results.forEach(country => console.log(JSON.parse(JSON.stringify(country))));
+    console.log(`
+        `);
+  });
+  connection.end();
+}
+
+function three() {
+  connection.connect();
+  connection.query(
+    'SELECT name, population FROM cities WHERE population BETWEEN 500000 AND 1000000',
+    function(error, results) {
+      if (error) throw error.message;
+      console.log(`
+        3- Cities with population in between 500,000 and 1 million:
+        `);
+      results.forEach(city => console.log(JSON.parse(JSON.stringify(city))));
+      console.log(`
+        `);
+    }
+  );
+  connection.end();
+}
+
+function four() {
+  connection.connect();
+  connection.query('SELECT name, region FROM countries WHERE region = "Europe" LIMIT 20', function(
+    error,
+    results
+  ) {
+    if (error) throw error.message;
+    console.log(`
+        4- Countries on the continent ‘Europe’:
```

Comment:
Same thing - this is not exactly what you return, you cap your results at 20

Reply:
done :+1: